frontend/test/metabase/scenarios/dashboard/actions-on-dashboards.cy.spec.js

```diff
@@ -0,0 +1,128 @@
+import {
+  restore,
+  queryTestDB,
+  resetTestTable,
+  getTableId,
+  waitForSyncToFinish,
+  fillActionQuery,
+} from "__support__/e2e/helpers";
+
+const DB_ID = 2;
+const TEST_TABLE = "scoreboard_actions";
+
+["mysql", "postgres"].forEach(dialect => {
+  describe(`Write Actions on Dashboards (${dialect})`, () => {
+    beforeEach(() => {
+      cy.intercept("/api/card/*").as("getModel");
+
+      cy.intercept("POST", "/api/dashboard/*/dashcard/*/execute").as(
+        "executeAPI",
+      );
+
+      cy.intercept("GET", "/api/dashboard/*").as("dashboardLoad");
+
+      resetTestTable({ type: dialect, table: TEST_TABLE });
+      restore(`${dialect}-writable`);
+      cy.signInAsAdmin();
+      cy.request("POST", `/api/database/${DB_ID}/sync_schema`);
+      cy.request("POST", `/api/database/${DB_ID}/rescan_values`);
+      waitForSyncToFinish(0, DB_ID);
```

Comment:
@kulyk had the excellent suggestion that we could maybe abstract these into a reusable helper function.

I want to think about whether this can/should be combined into the restore or reset helpers 🤔